src/components/UI/JSONSchemaForm/ArrayFieldItem/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { ArrayFieldTemplateItemType } from '@rjsf/utils';
+import { Box } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+import React from 'react';
+import styled from 'styled-components';
+
+import ArrayFieldItemActions from '../ArrayFieldItemActions';
+
+const ArrayFieldItemW = styled(Box)`
+  position: relative;
+
+  &::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    top: 10px;
+    bottom: 10px;
+    border-left: 1px solid ${({ theme }) => normalizeColor('border', theme)};
+  }
+`;
+
+const ArrayFieldItemActionsWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-shrink: 0;
+  width: 35px;
+  padding-top: 10px;
+`;
+
+const ArrayFieldItem: React.FC<ArrayFieldTemplateItemType> = ({
+  disabled,
+  index,
+  hasMoveDown,
+  hasMoveUp,
+  hasRemove,
+  hasToolbar,
+  readonly,
+  onDropIndexClick,
+  onReorderClick,
+  children,
+}) => {
+  return (
+    <ArrayFieldItemW direction='row'>
```

Comment:
Just a small suggestion - could we have a more descriptive name than `ArrayFieldItemW` ? I assumed the `W` stands for wrapper, but I wasn't sure if the abbreviation was intentional

Reply:
Thank you for noticing, it's definitely unintentional. Will fix it.